CCL.Importer/Proxies/MiscReplacer.cs

```diff
@@ -8,6 +8,8 @@ internal class MiscReplacer : Profile
     {
         public MiscReplacer()
         {
+            ShouldMapField = f => AutoMapperHelper.IsPublicOrSerialized(f);
```

Comment:
I think we can add this once in Mapper.Configure?

Reply:
It should be possible, I'll try it and if it works I'll push it here.